### ISSUE TITLE
error - sudo: command not found 

### ISSUE THREAD

**Issue by shawnhooper** [shivammathur/setup-php#688]
**Describe the bug**

Starting today (January 20, 2023) am unable to get this action to run.  Throws `sudo: command not found` errors.

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [X] `v2`
- [ ] `v1`

**Runners**
- [X] GitHub Hosted
- [ ] Self Hosted

**Operating systems**

Ubuntu 

**PHP versions**

8.1

**To Reproduce**
```jobs:
  phpunit:
    name: "PHPUnit Unit & Feature Tests w/ Code Coverage Enabled"
    runs-on: ubuntu-latest
    container:
      image: kirschbaumdevelopment/laravel-test-runner:8.1
    services:
      mysql:
        image: mysql:5.7
        env:
          MYSQL_ROOT_PASSWORD: password
          MYSQL_DATABASE: phpunit
        ports:
          - 33306:3306
        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
    steps:
      - uses: actions/checkout@v1
        with:
          fetch-depth: 1
      - name: Install composer dependencies
        run: |
          composer install --no-scripts
      - name: Prepare Laravel Application
        run: |
          mkdir -p .coverage-report/html
          php artisan passport:keys
      - name: Setup PHP with PCOV
        uses: shivammathur/setup-php@v2
        with:
          php-version: '8.1'
          ini-values: pcov.directory=app
          coverage: pcov
```

**Expected behavior**

**Screenshots/Logs**
![image](https://user-images.githubusercontent.com/2073284/213787948-bb0f01a5-901b-4888-b22a-6c00a2ffe811.png)

**Comment by shivammathur** [comment]
@shawnhooper Fixed in https://github.com/shivammathur/setup-php/commit/dd0115673c4a47a832dff844189234f0e9b3966e. It wIll be in the next release.

**Comment by Arkanius** [comment]
Oh, I had same problem here. Nice to hear that already exists a fix. I'll be waiting the release

**Comment by viktorruskai** [comment]
Thanks for the fix. 😊 @shivammathur When are you planning the release?

**Comment by anderson-flores** [comment]
Thanks for the fix! Next release will be soon?

**Comment by shivammathur** [comment]
Released the fix in [2.24.0](https://github.com/shivammathur/setup-php/releases/tag/2.24.0)
